src/static/riot/queues/management.tag

```diff
@@ -80,25 +80,23 @@
         </div>
         <div class="content">
             <form class="ui form" ref="form">
-                <div class="ui active tab" data-tab="details">
-                    <div class="required field">
-                        <label>Name</label>
-                        <input name="name" placeholder="Name" ref="queue_name" value="{ _.get(selected_queue, 'name', '') }">
-                    </div>
-                    <div class="field">
-                        <div class="ui checkbox">
-                            <label>Is Public?</label>
-                            <input type="checkbox" ref="queue_public">
-                        </div>
+                <div class="required field">
+                    <label>Name</label>
+                    <input name="name" placeholder="Name" ref="queue_name" value="{ _.get(selected_queue, 'name') }">
+                </div>
+                <div class="field">
+                    <div class="ui checkbox">
+                        <label>Is Public?</label>
```

Comment:
instead of "Is public?" lets do "Make Public" so that the purpose of the checkbox is more clear.

Reply:
✔️